deploy/template.yaml

```diff
@@ -1071,6 +1049,12 @@ Resources:
               KeyType: "HASH"
           Projection:
             ProjectionType: "ALL"
+        - IndexName: "accessToken-index"
```

Comment:
Namespace on index. Do we need it?

Reply:
Doesn't seem to be anything against this in the documentation, but I've tried it and it doesn't see to want to create the index. Also noticed if you try to create an index manually in the console it does add -index on the end

Reply:
Confirmed the namespace isn't needed, I've renamed to `accessToken`